frontend/src/css/components/blogCard.scss

```diff
@@ -3,6 +3,7 @@
 .blog-card {
   &__container {
     border-radius: 5px;
+    width: 340px;
```

Comment:
Do not give it a fixed with. Blog card will take the width of the parent element. 

```
<div className="cell large-4>
    <BlogCard />
</div>
```

Reply:
Okay, got it 🙌

Reply:
Okay, got it 👍